typescript/simple-mercari-web/src/App.tsx

```diff
@@ -7,16 +7,16 @@ function App() {
   // reload ItemList after Listing complete
   const [reload, setReload] = useState(true);
   return (
-    <div>
+    <div className= "App">
       <header className="Title">
         <p>
           <b>Simple Mercari</b>
         </p>
       </header>
-      <div>
+      <div className='Listing'>
```

Comment:
```suggestion
      <div>
```

But actually, I'll suggest using the classname directly inside the object! Maybe that's what you really want? See below for example.

Reply:
I fixed.